web/global/components/navigation/DefaultTopNav.js.jsx

```diff
@@ -32,6 +32,7 @@ class DefaultTopNav extends Binder {
   }
 
   componentWillMount() {
+    // will be depreciated in React 17+
     window.addEventListener('scroll', this._handleScroll);
   }
```

Comment:
i'm going to kill this as soon as i dig in there. methinks the UI needs to be stripped down out of the box

Reply:
+1